deb/etc/systemd/system/docker-metadata.service.d/docker-start-script.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+TARGET_FILE=/etc/systemd/system/docker-metadata.service.d/docker-metadata-environment
+
+function get_enabled(){
+  etcdctl get config/_global/proxy/enabled || echo "false"
+}
+
+# Clear environment file
+echo "" > "${TARGET_FILE}"
```

Comment:
Why do you clear the environment file? You can overwrite it with the new contents instead of appending the proxy settings to the previously cleared file.

Reply:
Maybe put this instruction into the else path of `if [ "$(get_enabled)" == "true" ]`.